framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -73,6 +73,7 @@
 import org.checkerframework.framework.util.typeinference.DefaultTypeArgumentInference;
 import org.checkerframework.framework.util.typeinference.TypeArgInferenceUtil;
 import org.checkerframework.framework.util.typeinference.TypeArgumentInference;
+import org.checkerframework.idesupport.TypeInformationPresenter;
```

Comment:
I don't think it's worth to have a new top-level package just for this single class. How about moving this to org/checkerframework/framework/util ?

Reply:
Agreed. Done.